t/basic.t

```diff
@@ -73,3 +73,6 @@ use Test2::Plugin::NoWarnings;
 }
 
 done_testing();
+
+# Make sure we respect done_testing()
+warn "This should not cause the test to fail!\n";
```

Comment:
We also need to make sure that the warning is not suppressed. I think the way to do this is to make _another_ test file that we execute via `prove` and then capture its output.